Classes/common/CDTDatastore+Conflicts.m

```diff
@@ -35,18 +35,16 @@ @implementation CDTDatastore (Conflicts)
 
 - (NSArray *)getConflictedDocumentIds
 {
-    if (![self.database open]) {
-        return nil;
-    }
-
-    return [self.database getConflictedDocumentIds];
+    // This property is not synthesized, it is a method that already ensures that the
+    // database is open (or return nil)
+    return (self.database ? [self.database getConflictedDocumentIds] : nil);
```

Comment:
Looks like I wrote this silliness..

However, given this: https://github.com/cloudant/CDTDatastore/blob/master/Classes/common/CDTDatastore.m#L79-L84.

I think we could just have 
```
- (NSArray *)getConflictedDocumentIds
 {
    return [self.database getConflictedDocumentIds];
}
```